app/scripts/first-time-state.js

```diff
@@ -9,17 +9,16 @@
  */
 const initialState = {
   config: {},
-  PreferencesController: {
-    frequentRpcListDetail: [
-      {
+  NetworkController: {
```

Comment:
I guess this is passed through migrations so perhaps this shouldn't (or doesn't need to) change?